website/cypress/contract/oasst_api_contract_tests.cy.ts

```diff
@@ -1,4 +1,4 @@
-import { OasstApiClient, OasstError } from "src/lib/oasst_api_client";
+import { OasstApiClient, OasstError } from "../../src/lib/oasst_api_client";
```

Comment:
I'd really like to avoid relative imports.  What requires this relative import here?